runners/direct-java/src/test/java/org/apache/beam/runners/direct/KeyedPValueTrackingVisitorTest.java

```diff
@@ -111,7 +113,13 @@ public void unkeyedInputWithKeyPreserving() {
                                 KvCoder.of(StringUtf8Coder.of(), VarIntCoder.of()))))));
 
     PCollection<KeyedWorkItem<String, KV<String, Integer>>> unkeyed =
-        input.apply(ParDo.of(new ParDoMultiOverrideFactory.ToKeyedWorkItem<String, Integer>()));
+        input
+            .apply(ParDo.of(new ParDoMultiOverrideFactory.ToKeyedWorkItem<String, Integer>()))
+            .setCoder(
```

Comment:
I may have been lazy in not letting this be automated, since it never came up. It is true that `KeyedWorkItem` is not in the default registry, but I think `getDefaultOutputCoder` should be able to provide a coder. Feel free to file a JIRA against me for this.